src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/PerformanceTrackerService.cs

```diff
@@ -61,6 +62,37 @@ private PerformanceQueue GetQueue(bool forSpanAnalysis)
 
         public void AddSnapshot(IEnumerable<AnalyzerPerformanceInfo> snapshot, int unitCount, bool forSpanAnalysis)
         {
+            foreach (var perfInfo in snapshot)
```

Comment:
I presume the intent for this PR to add aggregate logging for these snapshots, and a later PR will then delete our existing code that does custom aggregation of these snapshots and logs them in background?

Reply:
Our existing logging support in this area was a bit too complicated for me to tackle to start with. I *feel* like the old support can be mostly removed, but it's something I would like to discuss further with you outside this PR.